{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "396e8bb2_100f4047",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1115919
      },
      "writtenOn": "2021-02-05T06:17:02Z",
      "side": 1,
      "message": "PTAL.\nThe failure looks independent from this CL. I\u0027ll double check it later (rerunning dry-run now).",
      "revId": "b8a4af56a7bba039264f4f9e152233a704388e62",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}